test/common/ssl/ssl_socket_test.cc

```diff
@@ -236,9 +242,12 @@ const std::string testUtilV2(
   client_connection->addConnectionCallbacks(client_connection_callbacks);
   client_connection->connect();
 
-  size_t counter = 0;
-  auto stopSecondTime = [&]() {
-    if (++counter == 2) {
+  size_t connect_count = 0;
+  auto connect_second_time = [&connect_count, &dispatcher, &server_connection, &client_connection,
```

Comment:
Actually, I agree it's a bit ridiculous to have explicit capture lists in tests when it gets this long. My rule of thumb is in production code, make it explicit (avoid unintended mistakes that can creep in) and in test code, make the capture list explicit when it's short, otherwise you can wildcard it. I had though in the example I pointed at it would only be two items long, but I guess that's not the case across this file.

Reply:
Should I revert it or leave it as-is, then? I'm fine with either.

Reply:
Yeah, if you could revert back the ones that are actually ridiculous (and leave the shorter ones as is) that would be great.